cookbook/controller/service.rst

```diff
@@ -232,6 +234,34 @@ inject *only* the exact service(s) that you need directly into the controller.
    are valid, exactly how you want to organize your reusable code is up to
    you.
 
+Base Controller Methods and their Service Replacements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This table explains how to replace the convenience methods of the base
+controller.
+
++-----------------------------+------------------------------------+----------------------------------------------------------------+
+| Method                      | Service                            |  PHP Code                                                      |
++=============================+====================================+================================================================+
+| ``createForm``              | ``form.factory``                   | ``$formFactory->create($type, $data, $options)``               |
```

Comment:
there is no `getForm()` here:
https://github.com/symfony/symfony/blob/2.3/src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php#L164